### PR TITLE
let phpstan-ga install dependencies

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

**Comment by dbu** [comment]
waiting for https://github.com/OskarStark/phpstan-ga/pull/50 (and https://github.com/OskarStark/phpstan-ga/pull/49 to see the actual problem directly)

**Comment by dbu** [comment]
worked around the problems. we can merge this

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace Rokka\RokkaClientBundle\DependencyInjection;
 
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
```
Comment (chregu) [inline src/DependencyInjection/Configuration.php]: That's not needed aynmore, right?

Comment (dbu) [reply src/DependencyInjection/Configuration.php]: yep, removed.

seems that the cs fixer is also not running anymore. we could add it to the static.yml workflow. no time right now though.

**Pull request merged**